.chloggen/handle-conflits.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
typo in filename, should be "conflicts"

Reply:
this file is deleted at release time and its contents added to the changelog, it is ok to merge with a typo.